openhtf/io/output/__init__.py

```diff
@@ -1,26 +1,41 @@
-# Copyright 2014 Google Inc. All Rights Reserved.
+"""Output module."""
```

Comment:
"""
This module contains support for various built-in output mechanisms.

Here, a base OutputToFile class is implemented to provide simple output to
a file via the pickle serialization mechanism.  It can be subclassed to implement
alternative serialization schemes, see json_factory.py and mfg_inspector.py for
examples.
"""